Jenkinsfile

```diff
@@ -154,11 +154,13 @@ helpers.rootLinuxNode(env, {
                         // Install kbfsfuse first so we can start on dockerizing.
                         sh "go install github.com/keybase/kbfs/kbfsfuse"
                         sh "cp ${env.GOPATH}/bin/kbfsfuse ./kbfsfuse/kbfsfuse"
+                        sh "go install github.com/keybase/kbfs/kbfsgit/git-remote-keybase/git-remote-keybase"
```

Comment:
This should be `go install github.com/keybase/kbfs/kbfsgit/git-remote-keybase`

Reply:
oooh right!